service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1617,20 +1619,24 @@ components:
 
     HintQuery:
       type: object
-      description: Get display hints
+      description: |
+        Get display hints. Only one of relatedEntity and entityQuery may be provided
       properties:
         relatedEntity:
           description: |
             Optional related entity.
-            If specified, then we return hints computed across related instances (e.g. condition occurrences where condition id=11).
-            Otherwise, we return hints computed across all instances (e.g. all persons).
+            If specified, then we return hints computed across related instances 
+            (e.g. condition occurrences where condition id=11). Otherwise, 
+            we return hints computed across all instances (e.g. all persons).
           type: object
           properties:
             name:
               type: string
               description: Related entity name
             id:
               $ref: "#/components/schemas/Literal"
+        entityQuery:
```

Comment:
You could shorten this to `query`. There isn't anything else for it to refer to.

Reply:
sure.